### PR TITLE
doc: add ceph-kvstore-tool's man

### PR DESCRIPTION
**Pull request by Liuchang0812**
Signed-off-by: liuchang0812 <liuchang0812@gmail.com>

### CODE REVIEW

File: doc/man/8/ceph-kvstore-tool.rst [hunk 1]
```diff
@@ -0,0 +1,79 @@
+orphan:
```
Comment (tchaikov) [inline doc/man/8/ceph-kvstore-tool.rst]: should be
```
:orphan:
```

File: doc/man/8/ceph-kvstore-tool.rst [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+which are as follows:
+
+:command:`list [prefix]`
+    Print all keys that belong to the prefix.
+
+:command:`list-crc [prefix]`
+    Print all crc that belong to the prefix.
+
+:command:`exists <prefix> [key]`
+    Query whether prefix or key is exist.
```
Comment (tchaikov) [inline doc/man/8/ceph-kvstore-tool.rst]: "exist" is a verb, not an adj.

might want to put:

> Check if there is any KV pair stored with the URL encoded prefix. If `key` is also specified, check for the `key` with the `prefix` instead.

File: doc/man/8/ceph-kvstore-tool.rst [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+=====================================================
+ ceph-kvstore-tool -- ceph kvstore manipulation tool
+=====================================================
+
+.. program:: ceph-kvstore-tool
+
+Synopsis
+========
+
+| **ceph-kvstore-tool** <leveldb|rocksdb|bluestore-kv> <store path> command [args...]
```
Comment (tchaikov) [inline doc/man/8/ceph-kvstore-tool.rst]: use
```
*command*
```

as it is a metavar.

File: doc/man/8/ceph-kvstore-tool.rst [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+
+Description
+===========
+
+:program:`ceph-kvstore-tool` is a kvstore manipulation tool. It allows users to manipule
+leveldb/rocksdb's data(likes OSD's omap) offline.
```
Comment (tchaikov) [inline doc/man/8/ceph-kvstore-tool.rst]: add a space before '('.

s/likes/like/

File: doc/man/8/ceph-kvstore-tool.rst [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+
+Commands
+========
+
+:program:`ceph-kvstore-tool` utility uses many commands for administration purpose
+which are as follows:
```
Comment (tchaikov) [inline doc/man/8/ceph-kvstore-tool.rst]: s/administration/debugging/

**Comment by Liuchang0812** [comment]
```
Using context: make check (arm64)
Could not update commit status of the Pull Request on GitHub.
java.net.SocketTimeoutException: connect timed out
```

@tchaikov thanks for your reviewing. updated

**Comment by Liuchang0812** [comment]
jenkins test this please

**Comment by Liuchang0812** [comment]
jenkins test this please

File: doc/man/8/ceph-kvstore-tool.rst [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+
+:command:`rm <prefix> <key>`
+    Remove the KV pair stored with the URL encoded prefix and key.
+
+:command:`rm-prefix <prefix>`
+    Remove all KV pairs stored with the URL encoded prefix.
+
+:command:`store-copy <path> [num-keys-per-tx]`
+    Copy all KV pairs to another directory specified by ``path``. 
+    [num-keys-per-tx] is the number of KV pairs coped for a transaction.
```
Comment (amitkumar50) [inline doc/man/8/ceph-kvstore-tool.rst]: @Liuchang0812 Looks `copied`

Comment (Liuchang0812) [reply doc/man/8/ceph-kvstore-tool.rst]: thank you 👍

**Pull request merged**